aws/resource_aws_api_gateway_resource_test.go

```diff
@@ -65,6 +71,12 @@ func TestAccAWSAPIGatewayResource_update(t *testing.T) {
 						"aws_api_gateway_resource.test", "path", "/test_changed"),
 				),
 			},
+			{
+				ResourceName:      "aws_api_gateway_resource.test",
+				ImportState:       true,
+				ImportStateIdFunc: testAccAWSAPIGatewayResourceImportStateIdFunc("aws_api_gateway_resource.test"),
+				ImportStateVerify: true,
```

Comment:
given the ID doesn't change in the Read method, I don't think there's any value checking this here?